Proof read of the Soto 5 docs (#376)
soto-project/soto: documentation/S3 Multipart Upload.md

# S3 Multipart Upload

> S3 has a series of multipart upload operations. These can be used to upload an object to S3 in multiple parts. If your object is larger than 5GB you are required to use the multipart operations for uploading, but multipart also has the advantage that if one part fails to upload you don't need to re-upload the whole object, just the parts that failed. Finally you can upload multiple parts at the same time and thus improve your upload speed. The multiple part uploading isn't support by Soto at the moment, but it is on the list of improvements to make. You can read more about multipart upload [here](https://docs.aws.amazon.com/AmazonS3/latest/dev/mpuoverview.html).

Reviewer: Maybe we should add "You can read Amazon's documentation on multipart upload here"

Author: Do you mean something like

```suggestion
S3 has a series of multipart upload operations. These can be used to upload an object to S3 in multiple parts. If your object is larger than 5GB you are required to use the multipart operations for uploading, but multipart also has the advantage that if one part fails to upload you don't need to re-upload the whole object, just the parts that failed. Finally you can upload multiple parts at the same time and thus improve your upload speed. The multiple part uploading isn't support by Soto at the moment, but it is on the list of improvements to make. You can read  Amazon's documentation on multipart upload [here](https://docs.aws.amazon.com/AmazonS3/latest/dev/mpuoverview.html).
```

Reviewer: yep
